LDM-148.tex

```diff
@@ -444,6 +444,7 @@ \subsubsection{Archiving}\label{archiving}
 including the wavefront sensors and the guide sensors of the LSSTCam or
 ComCam when so configured, retrieving them from their respective Camera
 Data System instances.
+The Image Archiver service for at least LSSTCam will be implemented using the Camera Control System's image writing component as a result of the removal of crosstalk-corrected images.
```

Comment:
The “as a result...” is just capturing historical decision making processes; I'm not sure it adds anything to this document, and, as such, I would remove it.